test/airport.spec.js

```diff
@@ -0,0 +1,288 @@
+const { assertEquals, assertTrue, assertFalse } = require(`../framework`);
+const Airport = require(`../src/Airport`);
```

Comment:
Have written that in the README as well, but for some reason the airport.js file changed from capital A to small a, therefore this require will not work properly, on my PC it is still capital A, don't know why.

Reply:
Windows systems are case insensitive when it comes to file names - unix-based systems (Linux and MacOS) are generally case-sensitive.  Not sure why this has happened - I suspect Windows is doing too much!